Movie Challenge/Movie Challenge/Controllers/Favorite/Cells/FavoriteTableViewCell.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  FavoriteTableViewCell.swift
+//  Movie Challenge
+//
+//  Created by Igor Gomes Arantes on 11/09/2018.
+//  Copyright © 2018 igor gomes arantes. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class FavoriteTableViewCell : UITableViewCell, UICollectionViewDataSource, UICollectionViewDelegate{
```

Comment:
o organize better you have to implement the Delegates and DataSources inside extensions blocks.